cohort-engine-api-web/src/main/java/com/ibm/cohort/engine/api/service/CohortEngineRestConstants.java

```diff
@@ -0,0 +1,23 @@
+/*
+ * (C) Copyright IBM Corp. 2020, 2020
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ */
+package com.ibm.cohort.engine.api.service;
+
+public class CohortEngineRestConstants {
+	public static final String SERVICE_MAJOR_VERSION = "v1";
+	public static final String SERVICE_TITLE = "IBM Cohort Engine";
+	public static final String SERVICE_DESCRIPTION = "Service to evaluate cohorts and measures";
+
+	// Provide a set of package names for swagger to search for REST APIs (comma delimited).
+	// Static assignment example
+	//public static final String SERVICE_SWAGGER_PACKAGES =  "com.ibm.watson.common.service.package1, com.ibm.watson.common.service.package2";
+
+	// Dynamic assignment using base package name from this class
+	private static String className = CohortEngineRestConstants.class.getCanonicalName();
+	public static final String SERVICE_SWAGGER_PACKAGES = className.substring(0, className.lastIndexOf('.'));
+
+	public static final String DARK_LAUNCHED_NEW_DATE_API_FEATURE = "new_date_api_feature";
```

Comment:
Curious as to the purpose for these two parameters?

Reply:
The rest service framework provides a handy means for having REST API calls that can be dark launched via env vars. The mechanism includes hiding them from the OpenAPI/Swagger interface when they are disabled. They would be used on the REST API definition and in the env vars to enable/disable certain API calls. These were 2 leftover examples of that from the sample application which I will cleanup.